env.py

```diff
@@ -39,8 +45,8 @@ def get_env(args):
         uuid_tag = args.uuid
 
     if args.architecture == 'ram':
-        env = FrameStack(env, args.history_length, training=True)
-        test_env = FrameStack(test_env, args.history_length, training=False)
+        env = make_ram(env, args.history_length, training=True)
+        test_env = make_ram(test_env, args.history_length, training=False)
```

Comment:
This is unrelated to the branch, but when testing with architecture=='ram', I noticed that FrameSkip (of course) doesn't take a `training` parameter. But I felt that we wanted one anyway, so I made a `make_ram` function and wrapped it with `EpisodicLifeEnv` like the others.